libs/sdk-ui-dashboard/src/presentation/filterBar/filterBar/DefaultFilterBar.tsx

```diff
@@ -98,7 +98,7 @@ export function DefaultFilterBar(props: IFilterBarProps): JSX.Element {
 
     return (
         <DefaultFilterBarContainer>
-            <div className="dash-filters-date dash-filters-attribute">
```

Comment:
Is this safe? The original KD had these both as well (however weird it looks)

Reply:
It should be...  I updated css as well (it should be safe even for plugins, as there is only `position: relative` for this class)

Reply:
ok, thanks :)